core/src/test/java/net/sourceforge/jnlp/PluginBridgeTest.java

```diff
@@ -198,32 +198,32 @@ public void testEmbeddedJnlpWithValidCodebase() throws Exception {
 
         //Codebase within jnlp file is VALID
         /**
-        <?xml version="1.0"?>
-            <jnlp spec="1.5+"
-              href="EmbeddedJnlpFile.jnlp"
-              codebase="http://www.redhat.com"
-            >
-
-            <information>
-                <title>Sample Test</title>
-                <vendor>RedHat</vendor>
-                <offline-allowed/>
-            </information>
-
-            <resources>
-                <j2se version='1.6+' />
-                <jar href='EmbeddedJnlpJarOne.jar' main='true' />
-                <jar href='EmbeddedJnlpJarTwo.jar' main='true' />
-            </resources>
-
-            <applet-desc
-                documentBase="."
-                name="redhat.embeddedjnlp"
-                main-class="redhat.embeddedjnlp"
-                width="0"
-                height="0"
-            />
-           </jnlp>
+         <?xml version="1.0"?>
```

Comment:
please deactivate formatting of comments and convert this javadoc to a normal comment as it is not valid javadoc